recipe/meta.yaml

```diff
@@ -50,7 +53,9 @@ about:
   home: https://github.com/CycloneDX/cyclonedx-python-lib
   summary: A library for producing CycloneDX SBOM (Software Bill of Materials) files.
   license: Apache-2.0
-  license_file: LICENSE
+  license_file:
+    - LICENSE
+    - NOTICE
```

Comment:
:+1:

Reply:
yes, poetry is the blocker - https://github.com/python-poetry/poetry/issues/8441